src/renderer/LargeButton.tsx

```diff
@@ -0,0 +1,30 @@
+import './styles/LargeButton.scss';
+
+interface ILargeButtonProps {
+  children: JSX.Element;
+  ariaLabel: string;
+  isDisabled: boolean;
+  handleChange: () => void;
+}
+
+const LargeButton = ({
```

Comment:
Cool component! Something we should do later is refactor the `PrerequisiteModal` so that it uses this component too. We can add `width` and `height` as props to make the button more flexible.

Is there a reason why this is called `LargeButton` and not just `Button`?

Reply:
Tbh nope. I'll see if I can refactor this in.